client/package.json

```diff
@@ -66,36 +66,38 @@
   },
   "prettier": "@genesislcap/prettier-config",
   "devDependencies": {
-    "@genesislcap/design-system-configurator": "14.91.1",
-    "@genesislcap/foundation-testing": "14.91.1",
-    "@genesislcap/genx": "14.91.1",
+    "@genesiscommunitysuccess/cep-fast-plugin": "4.1.1",
```

Comment:
is this the line that enables LSP or there is a config somewhere else?

Reply:
Most of the enable functionality is in the base blank app seed which this builds upon. This PR mainly fixes issues in code and config from this seed specific files

Reply:
Because this seed doesn't currently alter the `tsconfig.json` which contains the config, that file is jsut omitted from this diff as it isn't needed